openapi/api.yml

```diff
@@ -85,9 +85,9 @@ paths:
                 format: binary
                 description: The file content in a binary format (e.g. XLSX).
 
-  /async_download:
+  /trigger_async_download:
     get:
```

Comment:
This feels much better as a POST request - since it's doing a chunk of work and we need to pass through sensitive information like email addresses. With GET requests, all parameters are passed in the URL, which will mean they get logged and persisted. PII/etc shouldn't be passed through query params.

I know it's not strictly part of this PR, but if we can either include it or fix this as a quick follow-up, it would be great.

Reply:
Changed it to a 'post'.